packages/gluestick/src/generator/templates/HomeApp.js

```diff
@@ -1,47 +1,44 @@
 /* @flow */
 import type { CreateTemplate } from '../../types';
 
-module.exports = (createTemplate: CreateTemplate) => {
-  const template = createTemplate`
-    import React, { Component } from "react";
-    import { bindActionCreators } from "redux";
-    import { connect } from "react-redux";
-    import Helmet from "react-helmet";
+module.exports = (createTemplate: CreateTemplate) => createTemplate`
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import Helmet from "react-helmet";
 
-    import Home from "components/Home";
+import Home from "../components/Home";
```

Comment:
We do but `component` alias does not make any sense. Is it shared components or components from which app?